internal/cri/server/container_image_mount.go

```diff
@@ -102,7 +102,11 @@ func (c *criService) mutateImageMount(
 	target := c.getImageVolumeHostPath(sandboxID, imageID)
 
 	// Already mounted in another container on the same pod
-	if stat, err := os.Stat(target); err == nil && stat.IsDir() {
+	mounted, err := ensureImageVolumeMounted(target)
```

Comment:
note to self requires .. "careful" rebase against..: https://github.com/containerd/containerd/pull/11578/files#diff-08bc007bc2a695705409801af29731a387ce26d53e8161dc01e69c7e7e3fa758R107

Reply:
Sure! Thanks!